signing_clients/apps.py

```diff
@@ -102,6 +105,21 @@ def _digest(data):
     return {'md5': md5.digest(), 'sha1': sha1.digest()}
 
 
+def get_addon_type_from_signature(pkcs7):
+    info = SignatureInfo(pkcs7)
+    ou = get_signer_organizational_unit_name(info)
+
+    if ou == constants.OU_PRODUCTION:
+        if info.signer_certificate['issuer']['common_name'] == 'firefox-hotfix@mozilla.org':
+            return 'hotfix'
+        return 'add-on'
+    elif ou == constants.OU_PRIVILEGED_WEB_EXTENSION:
+        return 'privileged web extension'
```

Comment:
During initial discussion, we made sure to stay away from the word `privileged`, which is why they are called **Mozilla** Extensions.